bin/pull-prod-admin-config

```diff
@@ -0,0 +1,147 @@
+#! /bin/bash
+
+# DOC: Dump program and question configuration from production or staging PostgreSQL database to a local archive file using pg_dump.
+
+set -e
+
+export AWS_DEFAULT_REGION=us-west-2
+TIMESTAMP=$(date +%s)
+ENVIRONMENT=staging
+STAGING_DB_PASSWORD_ID=arn:aws:secretsmanager:us-west-2:405662711265:secret:staging-databasepassword-rh1wfF
+PROD_DB_PASSWORD_ID=arn:aws:secretsmanager:us-west-2:405662711265:secret:prod-databasepassword-OM7x4q
+DB_PASSWORD_ID=$STAGING_DB_PASSWORD_ID
+
+if [ "$1" = "prod" ]; then
```

Comment:
for reasons, guide prefers [[ over [

https://google.github.io/styleguide/shellguide.html#test----and---

Reply:
done